src/test/java/com/exasol/projectkeeper/ProjectKeeperIT.java

```diff
@@ -189,6 +187,22 @@ void testChangesFileGeneration() throws IOException, GitAPIException, Verificati
                         containsString("* Updated `org.apache.maven.plugins:maven-surefire-plugin:2.12.4` to")));
     }
 
+    private void setupDemoProjectWithDependencyChange(final boolean released) throws IOException, GitAPIException {
+        final Git git = Git.open(this.projectDir.toFile());
+        writePomWithOneDependency("0.1.0", "1.0.0");
+        makeRelease(git, "0.1.0");
+        writePomWithOneDependency("0.2.0", "1.0.1");
+        if (released) {
+            makeRelease(git, "0.2.0");
```

Comment:
I think git doesn't have a 'release' concept, maybe we can rename the methods to addTag?

Reply:
:+1: